src/main/java/seedu/duke/model/task/ProjectBacklog.java

```diff
@@ -47,6 +55,10 @@ public void addTask(String title, String description, String priority) {
         backlogTasks.add(new Task(newTaskId, title, description, priority));
     }
 
+    public int idSize() {
```

Comment:
Got a getter for `nextId` already, at line 37

Reply:
Think remove it then